fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
Neat. How does it detect when the logs are JSONL? Look for { at beginning?
Try to parse it?

I don't object to extracting more metadata, but the actual original line
should be available as the standard field for log message (whatever
docker's "log" gets translated to normally).

On Wed, Sep 30, 2015 at 12:27 PM, Jimmi Dyson <notifications@github.com>
wrote:

> In fluentd/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/4#discussion_r40816527>
> :
>
> > @@ -135,18 +134,36 @@
> >      bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
> >      ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
> >    </filter>
> > +  <filter **kibana**>
>
> The kubernetes metadata plugin likes JSON logs because it (me) likes
> structured logs (events). Anything logged in JSON will be merged into the
> fluentd record so that fields will show up in Elasticsearch as separate
> fields rather than just a single string.
>
> If you want to disable it add merge_json_log false to the kubernetes
> filter plugin config.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/4/files#r40816527>
> .
>